guides/common/modules/proc_creating-objects-by-using-python.adoc

```diff
@@ -0,0 +1,118 @@
+[id="creating-objects-by-using-python"]
+= Creating objects by using Python
+
+This script connects to the {ProjectNameX} API and creates an organization, and then creates three environments in the organization.
+If the organization already exists, the script uses that organization.
+If any of the environments already exist in the organization, the script raises an error and quits.
+
+.Python 3 example
+[source, Python, subs="attributes"]
+----
+#!/usr/bin/python3
+
+import json
+import sys
+
+try:
+    import requests
+except ImportError:
+    print("Please install the python-requests module.")
+    sys.exit(-1)
+
+# URL to your {ProjectServer}
+URL = "https://{foreman-example-com}"
+FOREMAN_API = f"\{URL}/api/"
+KATELLO_API = f"\{URL}/katello/api/"
+POST_HEADERS = {'content-type': 'application/json'}
+# Default credentials to login to {ProjectX}
+USERNAME = "admin"
+PASSWORD = "changeme"
+# Ignore SSL for now
+SSL_VERIFY = False
+
+# Name of the organization to be either created or used
+ORG_NAME = "MyOrg"
+# Name for life cycle environments to be either created or used
+ENVIRONMENTS = ["Development", "Testing", "Production"]
+
+
+def get_json(location):
+    """
+    Performs a GET by using the passed URL location
+    """
+    r = requests.get(location, auth=(USERNAME, PASSWORD), verify=SSL_VERIFY)
+    return r.json()
+
+
+def post_json(location, json_data):
+    """
+    Performs a POST and passes the data to the URL location
+    """
+    result = requests.post(
+        location,
+        data=json_data,
+        auth=(USERNAME, PASSWORD),
+        verify=SSL_VERIFY,
+        headers=POST_HEADERS
+    )
+    return result.json()
+
+
+def main():
+    """
+    Main routine that creates or re-uses an organization and
+    life cycle environments.
+    If life cycle environments already
```

Comment:
Fix wording; or in general, improve comments?

Reply:
The comments could indeed use some love :) if you have the time

Reply:
https://github.com/theforeman/foreman-documentation/issues/3379